docs/Makefile

```diff
@@ -46,3 +46,34 @@ markdownlint:
 
 markdownlint-fix:
 	$(MDL_CMD) --entrypoint="markdownlint-cli2-fix" davidanson/markdownlint-cli2-rules:${MDL_DOCKER_VERSION} "**/*.md" 
+
+# Define the directories
+OPERATOR_DIR := operator
+CONFIG_DIR := $(OPERATOR_DIR)/config
+HELM_TEMPLATES_DIR := helm/chart/templates
+METRICS_OPERATOR_DIR := metrics-operator
+
+.PHONY: resources
+
+# New target to automate the steps for both operators
+resources: generate-manifests copy-manifests update-crd-docs
+
+generate-manifests:
+	@echo "Generating manifests..."
+	cd $(OPERATOR_DIR) && make generate && make manifests
```

Comment:
please use `make -C $(OPERATOR_DIR)` to call make command from sub-makefiles. same for metrics operator